generate-attribution/generate-attribution-file.js

```diff
@@ -338,6 +340,15 @@ async function populateVersionAndModuleFromDep(dependencies) {
     return finalDeps;
 }
 
+function generateAuthorizationHeader() {
+    const githubToken = await fsPromises.readFile("/secrets/github-secrets/token", 'utf8');
```

Comment:
Probably not the most elegant but just add a .stat call before hand to make sure the file exists. If it doesn't, just return an empty object. I think ideally wed pass this in as a cli arg or use an env var, but this is totally fine